cppapi/server/device_3.cpp

```diff
@@ -300,741 +340,645 @@ Tango::AttributeValueList_3* Device_3Impl::read_attributes_3(const Tango::DevVar
 // arguments:
 //		in :
 //			- names: The names of the attribute to read
-//			- aid : Structure with pointers for data to be returned to caller (several pointers acoording to
+//			- data : Structure with pointers for data to be returned to caller (several pointers acoording to
```

Comment:
We can take this opportunity to fix this typo in the comment:
```suggestion
//			- data : Structure with pointers for data to be returned to caller (several pointers according to
```

Reply:
Fixed in https://github.com/tango-controls/cppTango/pull/698/commits/ad8e9efddfda3f5e8a63c43ae8c92156c30367d7